web/views/note_view.ex

```diff
@@ -1,4 +1,8 @@
 defmodule Notox.NoteView do
   use Notox.Web, :view
 
+  def render_markdown(markdown) do
+    import Earmark
+    Earmark.to_html(markdown) |> raw
```

Comment:
Got it under control. There are functions for this like `put_change` and `fetch_field`!